addons/wp/stringtable.xml

```diff
@@ -679,12 +684,12 @@
         </Key>
         <Key ID="STR_Lambs_WP_Context_Main_DisplayName">
             <English>Lamber Danger Waypoints</English>
-            <Polish>LAMBS Danger Punkty trasy</Polish>
+            <Polish>Lamber Danger Waypoints</Polish>

```

Comment:
LAMBS is actually correct here :P looks like there is a misspelling in the English version in this case.

Reply:
Because I coulnd't find appropriate name for difference between "LAMBS Danger Waypoints" and "LAMBS Danger Search Waypoints", I copy-pasted them from english (tbh I didn't see that "Lamber")